app/controllers/catalog_controller.rb

```diff
@@ -92,9 +92,9 @@ def self.modified_field
 
       # Add property as searchable all fields box and individually
       if Document.properties[prop].behaviors.include?(:stored_searchable)
-        config.add_show_field solr_name(prop, :stored_searchable) unless Document.properties[prop]['advance_search']
+        config.add_show_field solr_name(prop, :stored_searchable) if Document.properties[prop]['showable'] || Document.properties[prop]['showable'].nil?
 
-        if Document.properties[prop]['advance_search'] || Document.properties[prop]['advance_search'].nil?
+        if Document.properties[prop]['basic_searchable'] || Document.properties[prop]['basic_searchable'].nil?
           config.add_search_field(prop) do |field|
```

Comment:
`config.add_search_field` does not control whether metadata is searchbox searchable, only if it gets a dropdown/specific field search (useful for advanced search I suppose).
Documentation on `add_search_field`: http://jessiekeck.com/customizing-blacklight/search_fields/

We need to mess with this block to only add metadata that is `basic_searchable` to the `all_fields` search field: https://github.com/OregonDigital/OD2/blob/master/app/controllers/catalog_controller.rb#L210-L217

Reply:
We also should look at what OTHER fields are added from `config.show_fields.values` before we rip out that statement and replace it with loops/joins and filters from `Document.properties` and others.